test/test-fs.c

```diff
@@ -3600,6 +3600,53 @@ TEST_IMPL(fs_exclusive_sharing_mode) {
 }
 #endif
 
+#ifdef _WIN32
+TEST_IMPL(fs_file_flag_no_buffering) {
+  int r;
+
+  /* Setup. */
+  unlink("test_file");
+
+  ASSERT(UV_FS_O_APPEND > 0);
+  ASSERT(UV_FS_O_CREAT > 0);
+  ASSERT(UV_FS_O_DIRECT > 0);
+  ASSERT(UV_FS_O_RDWR > 0);
```

Comment:
You could turn these into STATIC_ASSERTs in src/ or src/win (probably the latter because I suppose it's possible for a UNIX to not know about e.g. O_DIRECT.)

Reply:
I just meant these to be part of the run-time test, which is Windows-only. I started changing these to STATIC_ASSERTs after your comment, but then on second-thought maybe it's fine as is? Let me know.

Reply:
It's not strictly necessary but if some invariant is expected to always be true you might as well encode it as a STATIC_ASSERT in src/win rather than a run-time check in the test suite.